cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -251,7 +250,7 @@ class ReporterCommand : CliktCommand(
 
         reportOptions.forEach { (format, option) ->
```

Comment:
Couldn't you make this map also case-insensitive? But maybe this is rather for another PR.

Reply:
I was nicer to do than I thought, thanks to a new function in Kotlin 1.4, so done.